README.md

```diff
@@ -43,6 +55,31 @@ The plugin can also be configured to use multiple password files if required.
 </authConfigs>
 ```
 
+## Generating passwords using htpasswd
+You can use the [htpasswd](http://httpd.apache.org/docs/2.0/programs/htpasswd.html) program from Apache to manage your password file. You must use the -s option with htpasswd to force it to use SHA1 encoding for the password. So for example, you can use the following command to create a password file called "passwd" and put the password for the user "user" in it:
+
+```htpasswd -c -s passwd user```
+
+### htpasswd on Windows
+
+htpasswd is not available on windows, but there are plenty of websites that perform the encryption for free. Make sure you use the SHA1 algorithm.
```

Comment:
This seems like a very bad recommendation. Can we instead suggest downloading the apache zip files from http://www.apachehaus.com/cgi-bin/download.plx or https://www.apachelounge.com/download/. These are the recommendations from the [apache site](http://httpd.apache.org/docs/2.4/platform/windows.html#down), and no installation is necessary since they provide zip downloads...